src/components/card/Card.tsx

```diff
@@ -0,0 +1,73 @@
+import SmartLink from '@components/link/SmartLink'
+import { Action } from 'types/card-types'
+import { FC, ReactNode } from 'react'
+
+type Props = {
+  children?: ReactNode
+  header?: ReactNode // will be either a StaticImage or GatsbyImage
+  title?: string
+  subtitle?: string
+  additionalHeaderContent?: ReactNode | ReactNode[]
+  body?: ReactNode
+  actions?: Action | Action[]
+  transparentBorder?: boolean
+  transparentBody?: boolean
+}
+
+const Card: FC<Props> = (props) => {
+  const {
+    children,
+    header,
+    title,
+    subtitle,
+    additionalHeaderContent,
+    body,
+    actions,
+    transparentBorder = false,
+    transparentBody = false,
+  } = props
+
+  // creates the SmartLink component/s for action/s
+  const createActionLinks = (
+    actions: Action | Action[],
+  ): JSX.Element | JSX.Element[] => {
+    const createAction = (action: Action) => (
+      <SmartLink
+        className="inline-block bg-navy-700 hover:bg-navy-800 rounded text-white px-6 py-3 mb-1 font-medium tracking-wide"
+        href={action.url}
+      >
+        {action.label}
+      </SmartLink>
+    )
```

Comment:
**style:** move outside of createActionLinks, because it does not depend on the scope. This will make the function body smaller and allows to test `createAction` directly.

Reply:
I've moved `createAction` outside of the scope of `createActionLinks` function.